elementor/elementor
# Display helpful message during Envato Template Kit import

## PR Checklist



## PR Type
What kind of change does this PR introduce?


## Summary

* Add message to Template Kit import processntor supports these features out of the box.


## Description


This is a basic message that will display if a user attempts to import an Envato Template Kit via the Elementor import process. Elementor currently doesn't support the full range of features we expect (such as Recommended Plugins and Global Styles).

We can remove these messages when Elementor supports these features and we're able to make kit format compatible.

![image](https://user-images.githubusercontent.com/543073/119934352-fc2b6980-bfc8-11eb-9ef6-2d3e0ef7075f.png)




## Test instructions
This PR can be tested by following these steps:

* Upload an Envato Template Kit ZIP file
* Notice the message before the import process starts

## Quality assurance


Fixes #
